src/DataGrid.tsx

```diff
@@ -1100,16 +1105,18 @@ function DataGrid<R, SR, K extends Key>(
           ...style,
           gridTemplateRows: templateRows,
           '--rdg-header-row-height': `${headerRowHeight}px`,
-          '--rdg-row-width': `${totalColumnWidth}px`,
+          '--rdg-grid-inline-size': `${totalColumnWidth}px`,
           '--rdg-summary-row-height': `${summaryRowHeight}px`,
-          '--rdg-grid-height': `${
+          '--rdg-grid-block-size': `${
             max(totalRowHeight, clientHeight) +
             headerRowHeight +
             summaryRowsCount * summaryRowHeight
           }px`,
           ...getLayoutCssVars()
         } as unknown as React.CSSProperties
       }
+      dir={direction}
+      key={direction} // recreate grid as scroll position is not compatible b/w ltr and rtl
```

Comment:
It is actually an issue? seemed to work fine 🤔

Reply:
Scroll in the middle and toggle the checkbox

![image](https://user-images.githubusercontent.com/1452717/152032999-7ba8335e-062f-4f36-a708-597675d372db.png)

Reply:
nvm. we need to recreate the grid component and not div